docs/workshop.md

```diff
@@ -600,11 +600,11 @@ You will see the `PRODUCT_LIST_CACHE_DISABLE` environment variable, select the e
 
 and set the value to `1` and click on the **Apply** button.
 
-Now if you try to refresh the list of products in the Web App (by refreshing the page) or calling the `/products` endpoint of the Catalog API (see Lab 2) you should see the response time of your API taking multiples seconds again.
+Now if you try to refresh the list of products in the Web App (by refreshing the page) or calling the `/products` endpoint of the Catalog API (see Lab 2) you should see the response time increasing again due to CosmosDb query, plus the artificial time increase we added to identify easily cached vs non-cached.
 
-You can check the response time of the last request (e.g. GET `/products`) in the green box on the bottom left of the Web App.
+You can check the response time of the last request (e.g. GET `/products`) opening the DevTools of your browser (F12 for Chrome or Edge), and open the Network tab to check the `/products` endpoint time to respond :

```

Comment:
I am wondering if we should keep using the browser for testing or if we should move to an HTTP client (e.g. REST Client or Bruno) to allow testing all operations (e.g. GET and POST).